npc/common/src/main/java/org/screamingsandals/lib/npc/NPCImpl.java

```diff
@@ -265,6 +266,20 @@ private void sendSpawnPackets(PlayerWrapper player) {
                 .entities(Collections.singletonList(AdventureHelper.toLegacy(tabListName)))
                 .sendPacket(player);
 
+                new SClientboundSetPlayerTeamPacket()
+                .teamKey(AdventureHelper.toLegacy(tabListName))
+                .mode(SClientboundSetPlayerTeamPacket.Mode.CREATE)
+                .displayName(Component.empty())
```

Comment:
Before you ask, Component.empty() otherwise bungee create a Npc 67373782 above each npc.